floodviz/static/css/style.css

```diff
@@ -16,3 +27,66 @@ body {
 	font: 10px sans-serif;
 	stroke-width: 1;
 }
+
+.col-sm-6 {
+	min-width: 500px;
+}
+
+#hydrograph_text {
+	margin-left: 50px;
+	text-align: center;
+}
+
+#map_text {
+	text-align: center;
+	width: 500px;
+	margin-left: 100px;
+}
+
+#peakflow_text {
+	width: 800px;
+	text-align: center;
+	margin-left: 65px;
+}
+
+#map, #hydrograph {
+	margin-left: 20px;
+	margin-right: 20px;
+}
+
+#pic1, #pic2 {
+	width: 500px;
+	height: 400px;
+	margin-top: 50px;
+	margin-bottom: 50px;
+	border: 1px solid black;
+}
+
+.header, .subheader, .footer-icons {
+	text-align: center;
+	border: 1px solid black;
+}
+
+#textbox1 {
+	max-width: 1200px;
+}
+
+#textbox1 {
+	margin-top: 50px;
+}
+
+#pic2 {
+	margin-left: 100px;
+}
+
+.icon {
+	margin-top: 100px;
+	margin-bottom: 100px;
+	height: 25px;
+	width: 25px;
+}
+
+#icons {
```

Comment:
Not a big fan of having a class name and id name almost the same. How about we call this #icon-div or #icon-container? More descriptive too.